apps/web/src/components/GlobalCheckClaimStatus/index.tsx

```diff
@@ -31,48 +33,80 @@ const GlobalCheckClaimStatus: React.FC<React.PropsWithChildren<GlobalCheckClaimS
  *
  * TODO: Put global checks in redux or make a generic area to house global checks
  */
+const GITHUB_ENDPOINT = 'https://raw.githubusercontent.com/pancakeswap/airdrop-v3-users/master'
+
 const GlobalCheckClaim: React.FC<React.PropsWithChildren<GlobalCheckClaimStatusProps>> = ({ excludeLocations }) => {
+  const { address: account } = useAccount()
+  const { chainId } = useActiveChainId()
+  const { pathname } = useRouter()
   const hasDisplayedModal = useRef(false)
   const { toastSuccess } = useToast()
   const { t } = useTranslation()
-  const [canClaimAnniversaryPoints, setCanClaimAnniversaryPoints] = useState(false)
-  const { claimAnniversaryPoints } = useAnniversaryAchievementContract()
-  const { canClaim } = useAnniversaryAchievementContract(false)
+  const [canClaimReward, setCanClaimReward] = useState(false)
+  const { claim } = useV3AirdropContract()
+  const { isClaimed } = useV3AirdropContract(false)
   const { fetchWithCatchTxError } = useCatchTxError()
-  const [onPresentAnniversaryModal] = useModal(
-    <AnniversaryAchievementModal
+
+  const { data } = useSWRImmutable('/airdrop-json', async () => {
+    const [feResponse, scResponse, merkleProofsResponse] = await Promise.all([
+      fetch(`${GITHUB_ENDPOINT}/forFE.json`),
+      fetch(`${GITHUB_ENDPOINT}/forSC.json`),
+      fetch(`${GITHUB_ENDPOINT}/v3MerkleProofs.json`),
+    ])
+    const [v3WhitelistAddress, v3ForSC, v3MerkleProofs] = await Promise.all([
+      feResponse.json(),
+      scResponse.json(),
+      merkleProofsResponse.json(),
+    ])
+
+    return {
+      v3WhitelistAddress,
+      v3ForSC,
+      v3MerkleProofs,
+    }
+  })
+
+  const [onPresentV3AirdropModal, closeV3AirdropModal] = useModal(
+    <V3AirdropModal
+      data={data?.v3WhitelistAddress[account] as WhitelistType}
```

Comment:
we have to use lowercase if you use github as static file

Reply:
Fixed.Nice catch